modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -517,14 +537,35 @@ This request returns a response similar to the following:
 [source,json,linenums]
 ----
 [
- {"timeStamp":"2021-02-10T00:00:00Z","objectStoreRequestCount":1},
- {"timeStamp":"2021-03-10T00:00:00Z","objectStoreRequestCount":15},
- {"timeStamp":"2021-04-10T00:00:00Z","objectStoreRequestCount":2}
+ {"timeStamp":"2023-02-10T00:00:00Z","objectStoreRequestCount":1},
+ {"timeStamp":"2023-03-10T00:00:00Z","objectStoreRequestCount":15},
+ {"timeStamp":"2023-04-10T00:00:00Z","objectStoreRequestCount":2}
 ]
 ----
 
+To view Object Store usage statistics for one day for a sub-organization,
+submit a `curl` command similar to the following, replacing
+ORG_ID with the organization ID for the sub-organization and
+BEARER_TOKEN with your bearer token:
+
+[source,console,linenums]
+----
+curl -X GET \
+  'https://object-store-stats.anypoint.mulesoft.com/api/v1/organizations/ORG_ID?startDate=2023-12-01T17%3A51%3A54.000Z&endDate=2023-12-01T17%3A51%3A54.000Z&period=1month' \
```

Comment:
We can add a startdate and enddate of 3 days for ex Dec 1 to Dec 3 and the period=1day in the query.